src/Connection/Http1Connection.php

```diff
@@ -276,16 +276,22 @@ private function readResponse(
 
         $parser = new Http1Parser($request, $bodyCallback, $trailersCallback);
 
-        $start = getCurrentTime();
+        $start = Loop::now();
+        $timeout = $request->getInactivityTimeout();
+        if ($timeout <= 0) {
+            $timeout = self::MAX_KEEP_ALIVE_TIMEOUT * 1000;
```

Comment:
Why this fallback instead of letting the user disable the timeout entirely?

Reply:
Meant to change – was being lazy so I could always use `Promise\timeout()`.